src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1PlacementApplicationDomainEventServiceTest.kt

```diff
@@ -193,6 +194,8 @@ class Cas1PlacementApplicationDomainEventServiceTest {
             assertThat(it.crn).isEqualTo(CRN)
             assertThat(it.nomsNumber).isEqualTo(application.nomsNumber)
             assertThat(it.occurredAt).isWithinTheLastMinute()
+            assertThat(it.metadata.containsKey(MetaDataName.CAS1_PLACEMENT_APPLICATION_ID))
+            assertThat(it.metadata[MetaDataName.CAS1_PLACEMENT_APPLICATION_ID].toString() == it.applicationId.toString())
```

Comment:
we'll need a second function call to apply an assertion e.g. containsEntry(key,value)